docs/call-numbers.md

```diff
@@ -20,7 +20,26 @@ Expected time: 5.5 - 6 hours.
 
 When browse lists fail to generate and would like to run the rake task before it is scheduled:
 
-ssh to `catalog-indexer1`. Currently the call_numbers task is deployed to run on catalog-indexer1 check the crontab to confirm. If not then check `catalog-indexer2` and `catalog-indexer3`.
-create a backup of the `tmp/orangelight_name_titles.sorted` file. Count the lines: `wc -l /tmp/orangelight_name_titles.sorted`.
-While on the right box, create a tmux session and run the call number rake task (`cd /opt/orangelight/current && RAILS_ENV=production SOLR_URL=http://lib-solr-prod4.princeton.edu:8983/solr/catalog-production bundle exec rake browse:call_numbers --silent >> /tmp/cron_log.log 2>&1`). It takes some time. When it finishes successfully count the lines in the file `wc -l /tmp/orangelight_name_titles.sorted` to ensure that there is not a big difference with the backup file which is located in `/home/deploy/orangelight_name_titles.sorted` or the most recent file which is located in `/tmp/orangelight_name_titles.sorted`.
-Run `cd /opt/orangelight/current && RAILS_ENV=production SOLR_URL=http://lib-solr8-prod.princeton.edu:8983/solr/catalog-production bundle exec rake browse:load_call_numbers --silent >> /tmp/cron_log.log 2>&1`.
+ssh to `catalog-indexer1`.
+
+Currently the call_numbers task is deployed to run on `catalog-indexer1` check the crontab to confirm. If not then check `catalog-indexer2` and `catalog-indexer3`.
+
+Create a backup of the `tmp/orangelight_name_titles.sorted` file.
+
+Count the lines: `wc -l /tmp/orangelight_name_titles.sorted`.
+
+While on the right box, create a tmux session and run the `call_numbers` rake task:
+
+```
+cd /opt/orangelight/current
+RAILS_ENV=production SOLR_URL=http://lib-solr-prod4.princeton.edu:8983/solr/catalog-production bundle exec rake browse:call_numbers --silent >> /tmp/cron_log.log 2>&1
+```
+
+It takes some time. When it finishes successfully count the lines in the file `wc -l /tmp/orangelight_name_titles.sorted` to ensure that there is not a big difference with the backup file which is located in `/home/deploy/orangelight_name_titles.sorted` or the most recent file which is located in `/tmp/orangelight_name_titles.sorted`.
```

Comment:
It takes around 3 hours. When it finishes successfully count the lines in the file `wc -l /tmp/call_number_browse_s.sorted` to ensure that there is not a big difference with the backup file which is located in `/home/deploy/call_number_browse_s.sorted` or the most recent file which is located in `/tmp/orangelight_name_titles.sorted`.